File: 01 - Multiples of 3 and 5/Python/main.py
inputValue = 1000

multiples = []
total = 0

number = 0

while number < inputValue:
    if number % 3 == 0 or number % 5 == 0:
        multiples.append(number)
    number += 1

for number in multiples:
    total += number

print('The answer is: ', total)

File: 01 - Multiples of 3 and 5/Go/main.go
package main

import "fmt"

func main() {

	var number int = 1000
	var result int = 0

	for i := 1; i < number; i++ {
		if (i % 3) == 0 {
			result = result + i
		} else if (i % 5) == 0 {
			result = result + i
		}
	}

	fmt.Println(result)

}
